_posts/2021-07-15-AI-fairness/AI_fairness_metrics.Rmd

```diff
@@ -0,0 +1,415 @@
+---
+title: "Starting to think about AI Fairness"
+description: > 
+  The topic of AI fairness metrics is as important to society as it is confusing. Confusing it is due to a number of reasons: terminological proliferation, abundance of formulae, and last not least the impression that everyone else seems to know what they're talking about. This text hopes to counteract some of that confusion by starting from a common-sense approach of contrasting two basic positions: On the one hand, the assumption that dataset features may be taken as reflecting the underlying concepts ML practitioners are interested in; on the other, that there inevitably is a gap between concept and measurement, a gap that may be bigger or smaller depending on what is being measured. In contrasting these fundamental views, we bring together concepts from ML, legal science, and political philosophy.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanafairnessmetrics
+date: 07-15-2021
+bibliography: bibliography.bib
+categories:
+  - R
+  - Concepts
+  - Meta
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpg
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+If you use deep learning for unsupervised part-of-speech tagging of
+Sanskrit [^1], or knowledge discovery in physics [^2], you probably
+don't need to worry about model fairness. If you're a data scientist
+working at a place where decisions are made about *people*, however, or
+an academic researching models that will be used to such ends, chances
+are that you've already been thinking about this topic. Or feeling that
+you should. And thinking about this is hard.
+
+[^1]: @Srivastava
+
+[^2]: @Cranmer
+
+It is hard for several reasons. In this text, I will go into *just one*.
+
+## The forest for the trees
+
+Nowadays, it is hard to find a modeling framework that does *not*
+include functionality to assess fairness. (Or is at least planning to.)
+And the terminology sounds so familiar, as well: "calibration",
+"predictive parity", "equal true [false] positive rate"... It almost
+seems as though we could just take the metrics we make use of anyway
+(recall or precision, say), test for equality across groups, and that's
+it. Let's assume, for a second, it really were that simple. Then the
```

Comment:
```suggestion
it. Let's assume, for a second, it really was that simple. Then the
```

Reply:
oop, that's surprising ... is this by gut feeling or is there even a rule/heuristic applicable here?